timeout_config.cc

```diff
@@ -9,9 +9,34 @@
 
 #include "timeout_config.hh"
 #include "db/config.hh"
+#include "db/timeout_clock.hh"
+#include <seastar/core/future.hh>
 
 using namespace std::chrono_literals;
 
+timeout_config::update_helper::update_helper(const utils::config_file::named_value<uint32_t>& source,
+                                             db::timeout_clock::duration& dest)
+    : _source(source)
+    , _action(std::make_unique<serialized_action>([this] { update(); return make_ready_future(); }))
```

Comment:
Somewhere on stop there should be call to serializeable_action::join() to make sure no outstanding updates are scheduler, but I don't see any

Reply:
this does not matter anymore. since i just use the updateable_value instead of the observer + action machinery for updating a local duration value.